plugins/GSdx/GSLocalMemory.cpp

```diff
@@ -41,7 +41,7 @@
 		for(int x = _r.left; x < _r.right; x += w >> 3) \
 		{ \
 			const uint8* src = BlockPtr(_base + off->block.col[x]); \
-			uint8* dst = &_dst[x * bpp]; \
+			dst = &_dst[x * bpp]; \
```

Comment:
Hum it will work because `dst` is only assigned once (outside of the loop context). However it would change the dst pointer.
Here a print of dst before the loop and after the loop
```
before f06d71c0
after f06d71c0
before f06d7cc0
after f06d7cc0
before f06d7dc0
after f06d7dc0
```
The same print with your change
```
before f06551c0
after f0655bc0
before f0655cc0
after f0655cc0
before f0655dc0
after f0656630
```